Aptoide/aptoide-client-v8
# [monetisation/ASV-1187] IronSource Interstitial

**What does this PR do?**

   Implements IronSource interstitials that should appear when the user clicks Install on AppView. Per specification, it is under an AB test (90% of the users will see it, while 10% will not). Analytics for ad impressions and clicks are also implemented. A user property is also added ("ab_test_ironsource") to help analyze data.

**Note:** IronSourceAdRepository is not being injected into AdsManager because IronSource needs access activity lifecycle, which means it cannot be provided in ApplicationModule (and AdsManager needs to be in ApplicationModule).

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] IronSourceInterstitialAdExperiment.java
- [ ] IronSourceAdRepository.java

**How should this be manually tested?**

  Test if interstitials are retrieved and shown with the AB test working (check Charles), as well as the analytics being sent. Also check if the facebook user property "ab_test_ironsource" is being set.


**What are the relevant tickets?**

  [ASV-1187](https://aptoide.atlassian.net/browse/ASV-1187)

**Code Review Checklist**
